.github/workflows/publish-apple-darwin-wasm.yml

```diff
@@ -45,13 +43,21 @@ jobs:
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
+      - name: Download artifact
+        uses: dawidd6/action-download-artifact@v2

```

Comment:
Can we use a github official action or not? https://github.com/actions/download-artifact

Reply:
we tried, but official one does not work cross workflows as seen on 
https://github.com/actions/download-artifact/issues/172